TESTS/mbedtls/config/aes_test.cpp

```diff
@@ -0,0 +1,547 @@
+        0x92, 0x29, 0x31, 0x9C, 0x19, 0xF1, 0x5B, 0xA4
+    },
+    {
+        0x05, 0x8C, 0xCF, 0xFD, 0xBB, 0xCB, 0x38, 0x2D,
+        0x1F, 0x6F, 0x56, 0x58, 0x5D, 0x8A, 0x4A, 0xDE
+    }
+};
+
+static const unsigned char aes_test_ecb_enc[3][16] = {
+    {
+        0xC3, 0x4C, 0x05, 0x2C, 0xC0, 0xDA, 0x8D, 0x73,
+        0x45, 0x1A, 0xFE, 0x5F, 0x03, 0xBE, 0x29, 0x7F
+    },
+    {
+        0xF3, 0xF6, 0x75, 0x2A, 0xE8, 0xD7, 0x83, 0x11,
+        0x38, 0xF0, 0x41, 0x56, 0x06, 0x31, 0xB1, 0x14
+    },
+    {
+        0x8B, 0x79, 0xEE, 0xCC, 0x93, 0xA0, 0xEE, 0x5D,
+        0xFF, 0x30, 0xB4, 0xEA, 0x21, 0x63, 0x6D, 0xA4
+    }
+};
+
+static const unsigned char aes_test_cbc_dec[3][16] = {
+    {
+        0xFA, 0xCA, 0x37, 0xE0, 0xB0, 0xC8, 0x53, 0x73,
+        0xDF, 0x70, 0x6E, 0x73, 0xF7, 0xC9, 0xAF, 0x86
+    },
+    {
+        0x5D, 0xF6, 0x78, 0xDD, 0x17, 0xBA, 0x4E, 0x75,
+        0xB6, 0x17, 0x68, 0xC6, 0xAD, 0xEF, 0x7C, 0x7B
+    },
+    {
+        0x48, 0x04, 0xE1, 0x81, 0x8F, 0xE6, 0x29, 0x75,
+        0x19, 0xA3, 0xE8, 0x8C, 0x57, 0x31, 0x04, 0x13
+    }
+};
+
+static const unsigned char aes_test_cbc_enc[3][16] = {
+    {
+        0x8A, 0x05, 0xFC, 0x5E, 0x09, 0x5A, 0xF4, 0x84,
+        0x8A, 0x08, 0xD3, 0x28, 0xD3, 0x68, 0x8E, 0x3D
+    },
+    {
+        0x7B, 0xD9, 0x66, 0xD5, 0x3A, 0xD8, 0xC1, 0xBB,
+        0x85, 0xD2, 0xAD, 0xFA, 0xE8, 0x7B, 0xB1, 0x04
+    },
+    {
+        0xFE, 0x3C, 0x53, 0x65, 0x3E, 0x2F, 0x45, 0xB5,
+        0x6F, 0xCD, 0x88, 0xB2, 0xCC, 0x89, 0x8F, 0xF0
+    }
+};
+
+int mbedtls_aes_ecb_test_mbedos(int verbose)
+{
+    /*
+     * ECB mode
+     */
+    int ret;
+    int u;
+    int mode;
+    unsigned int keybits;
+    unsigned char key[32];
+    unsigned char buf[64];
+    mbedtls_aes_context ctx;
+    memset(key, 0, 32);
+    mbedtls_aes_init(&ctx);
+    const unsigned char *aes_tests;
+    for (int i = 0; i < 6; i++) {
+        u = i >> 1;
+        keybits = 128 + u * 64;
+        mode = i & 1;
+
+        if (verbose != 0)
+            printf("  AES-ECB-%3d (%s): ", keybits,
+                   (mode == MBEDTLS_AES_DECRYPT) ? "dec" : "enc");
+
+        memset(buf, 0, 16);
+
+        if (mode == MBEDTLS_AES_DECRYPT) {
+            ret = mbedtls_aes_setkey_dec(&ctx, key, keybits);
+            aes_tests = aes_test_ecb_dec[u];
+        } else {
+            ret = mbedtls_aes_setkey_enc(&ctx, key, keybits);
+            aes_tests = aes_test_ecb_enc[u];
+        }
+
+        /*
+         * AES-192 is an optional feature that may be unavailable when
+         * there is an alternative underlying implementation i.e. when
+         * MBEDTLS_AES_ALT is defined.
+         */
+        if (ret == MBEDTLS_ERR_AES_FEATURE_UNAVAILABLE && keybits == 192) {
+            printf("skipped\n");
+            continue;
+        } else if (ret != 0) {
+            goto exit;
+        }
+
+        for (int j = 0; j < 10000; j++) {
+            ret = mbedtls_aes_crypt_ecb(&ctx, mode, buf, buf);
```

Comment:
Need to skip in case of `MBEDTLS_ERR_PLATFORM_FEATURE_UNSUPPORTED`

Reply:
Will change to match the previously referenced PR